packages/components/src/systems/ScreeningAlerts/ScreeningAlert.tsx

```diff
@@ -0,0 +1,130 @@
+import { createSignal, For } from 'solid-js';
+import Icon from '../../particles/Icon';
+import Text from '../../particles/Text';
+
+export interface ScreeningAlertType {
+  description: string;
+  type: string;
+  severity: string;
+  involvedEntities: { id: string; name: string; type: string }[];
+}
+
+/**
+ * Helper function to capitalize the severity
+ */
+const getSeverityText = (severity: string) => {
+  return severity.charAt(0).toUpperCase() + severity.slice(1);
+};
+
+/**
+ * Helper function to get class names for styling based off of the severity
+ */
+const getColorBySeverity = (
+  severity: string
+): { icon: string; background: string; border: string } => {
+  switch (severity) {
+    default:
```

Comment:
oh god I missed this the first time. This gives me the icks. TBH I didn't even know you could do this. Aren't switches evaluated linearly? 

At the very least can we put `minor` and `default` as the last in the switch.